update docker command to fit docker compose v2 and latest netbox release

Shanwer opened the pull request:
Docker Desktop continues to support a ```docker-compose``` alias to redirect commands to ```docker compose```, however, most production environments use Docker CLI instead which do not have this alias, and the official recommended command-line syntax is ```docker compose```.
The original version also has some mistakes like VERSION environment variable is undefined and wrong container netbox, which should be postgres to import these sql scripts. They are also included in PR #12 #11.

Shanwer committed:
docs(README.md): update docker command to fit docker compose v2 and latest netbox release

Novanz commented on README.md:
```diff
@@ -26,14 +26,22 @@ sudo -u postgres psql netbox < netbox-demo-data/sql/netbox-demo-$VERSION.sql
+docker cp netbox-demo-data/sql/netbox-demo-$VERSION.sql "$(docker compose ps -q postgres)":/tmp/netbox-demo.sql
+docker compose exec postgres bash -c "psql -U $POSTGRES_USER netbox < /tmp/netbox-demo.sql"
+# Delete imported demo sql
+docker compose exec postgres bash -c "rm /tmp/netbox-demo.sql"
```
I was preparing a similar PR. Since content of /tmp will be removed after a restart, i think the last line can be omitted.

Shanwer replied:
Yes, but these commands will be executed only once, in case they are never restarted, however both are ok I think.

grische commented on README.md:
```diff
@@ -26,14 +26,22 @@ sudo -u postgres psql netbox < netbox-demo-data/sql/netbox-demo-$VERSION.sql
+# Set 'YOUR_NETBOX_VERSION' to your current netbox version such as v4.3
+export VERSION=YOUR_NETBOX_VERSION
+# Stop netbox container before dropping database if it is running or it will shows 'ERROR: database "netbox" is being accessed by other users'.
+docker compose stop netbox
```
@Shanwer don't you also need to stop `netbox-worker`, assuming you are using netbox-docker's default compose file?

Shanwer replied:
It was tested and worked on my machine when I was interning, so I bet this is OK.

grische replied:
You were probably just lucky then.

The wiki also explicitely states that if you work with the database, you should turn off both:
https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting#:~:text=Restore%20that%20database

grische commented on README.md:
```diff
@@ -26,14 +26,22 @@ sudo -u postgres psql netbox < netbox-demo-data/sql/netbox-demo-$VERSION.sql
 # Load the demo data
-docker cp netbox-demo-$VERSION.sql "$(docker-compose ps -q netbox)":/opt/netbox/netbox/netbox-demo.sql
-docker-compose exec netbox bash -c "psql -U $POSTGRES_USER netbox < /opt/netbox/netbox/netbox-demo.sql"
+docker cp netbox-demo-data/sql/netbox-demo-$VERSION.sql "$(docker compose ps -q postgres)":/tmp/netbox-demo.sql
```
Maybe add a step that creates the sql file, by doing something like this:

```suggestion
wget -nv https://raw.githubusercontent.com/netbox-community/netbox-demo-data/refs/heads/master/sql/netbox-demo-$VERSION.sql -O netbox-demo.sql
docker cp netbox-demo.sql "$(docker compose ps -q postgres)":/tmp/netbox-demo.sql
rm netbox-demo.sql
```

The `docker compose` command will only work in the netbox-docker folder and having a netbox-demo-data subfolder is not guaranteed and not explained beforehand.
This new command will work independently if someone cloned the repo in the correct place or not.

Shanwer committed:
Apply suggestion from @grische

I assume user will clone this repo and having subfolder inside however this command would be better.

Co-authored-by: Grische <2787581+grische@users.noreply.github.com>

Shanwer committed:
stop netbox-worker when initial netbox database

Refer https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting#:~:text=Restore%20that%20database
